### ISSUE TITLE
How to create a trigger event on HTML (jQuery)

### ISSUE THREAD

**Issue by charlesp0096** [mathifold/mathifold#52]
Hello! I've uploaded a post lately that includes more than one "blue box" (the one that hides and shows text, for example, in demonstrations), but they all respond to the click at the same time, and I'd like them to work separately. Any ideas on how I could fix it? Creating from scratch a trigger event, or modifying the one existing...?

**Comment by juanjomadrigal** [comment]
Yes, that's true. Our system for these toggling boxes (eg. http://mathifold.org/en/lemma_derivative_exponential_origin.html - https://github.com/mathifold/mathifold/blob/master/_posts/en/2016-04-01-lem-derivative-exponential-origin.md) is quite clumsy at this moment, awkward for both content-makers and users. Some other system may be needed.

In any case, this post http://mathifold.org/en/theorem_derivatives_polynomials.html - https://github.com/mathifold/mathifold/blob/master/_posts/en/2016-03-01-thm-derivatives-polynomials.md uses several boxes. This may be helpful in the meanwhile.

**Comment by charlesp0096** [comment]
Thanks a lot! I believe it has helped me indeed to sort out my problem. I will check out if it has fixed my issue as soon as I send my next pull request.
